resources/assets/less/bem/contest.less

```diff
@@ -47,6 +48,52 @@
     }
   }
 
+  &__accordion {
+    background: #1A2223;
+    padding: @spacing * 2;
+  }
+
+  &__group {
+    margin: @spacing auto;
+    max-width: 600px;
+    background: #161a1b;
+    border: none;
+    .default-box-shadow();
+  }
+
+  &__group-heading {
+    display: flex;
+    justify-content: space-between;
+    padding: @spacing;
+
+    .white-link();
+    &:hover, &:visited, &:link, &:active {
+      .white-link();
+      text-decoration: none;
+    }
+
+    &[aria-expanded='true'] {
+      .@{_top}__section-toggle {
```

Comment:
I think it's easier to see behavior of an element when it's grouped together where it's declared.

```
.parent {
  // stuff
  &[state] {
    .el {
      // stuff which may or may not override base .el
    }
  }
}

.el {
  // stuff
}
```

vs

```
.parent {
  // stuff
}

.el {
  // stuff
  .parent[some-state] & {
      // stuff which may or may not override base .el
  }
}
```